### ISSUE TITLE
(MotoG4 play) Harpia current recovery (July 18) not working. Help offered

### ISSUE THREAD

**Issue by Cultura-Difusa** [Divested-Mobile/DivestOS-Build#135]
Hello there
Using Divestos version 17.1-20220710-dos-harpia. Began by trying a clean install with its concurrent recovery (divested-17.1-20220710-dos-harpia-recovery.img); installing said recovery did not work (all appeared ok on installation, but sent back to bootloader when restarting). At the moment I am using plain lineageOS/microG BOOTLOADER, and Divestos OS. Not perfect, since I cannot get "over the air updates" automatically installed.

I will be happy to test alternative recoveries on my phone for debugging purposes.
Any details you need I would be happy to offer.

Alternatively I could use a link with previous version of recovery, in order to be able to update system "over the air".

Thanks in advance.

**Comment by SkewedZeppelin** [comment]
yes something broke the recovery on harpia/surnia/osprey/merlin, the system itself does work however which makes it pretty strange.

just flash it with another recovery for now.

**Comment by SkewedZeppelin** [comment]
Given that you can't lock the bootloader on these devices using TWRP may be OK

**Comment by SkewedZeppelin** [comment]
https://github.com/Divested-Mobile/DivestOS-Build/commit/06254708be4afc24c3158c8c145adca35a58ccee may actually fix this
